webrtc.html

```diff
@@ -9942,14 +9942,14 @@ <h3>Mandatory To Implement Stats</h3>
         inherited dictionaries, and also attributes packetsReceived,
         bytesReceived, packetsLost, jitter, packetsDiscarded</li>
         <li>RTCInboundRTPStreamStats, with all required attributes from its
-        inherited dictionaries, and also attributes remoteId, framesDecoded</li>
+        inherited dictionaries, and also attributes receiverId, remoteId, framesDecoded</li>
         <li>RTCRemoteInboundRTPStreamStats, with all required attributes from
         its inherited dictionaries, and also attributes localId,
         roundTripTime</li>
         <li>RTCSentRTPStreamStats, with all required attributes from its
         inherited dictionaries, and also attributes packetsSent, bytesSent</li>
         <li>RTCOutboundRTPStreamStats, with all required attributes from its
-        inherited dictionaries, and also attributes remoteId, framesEncoded</li>
+        inherited dictionaries, and also attributes senderId, remoteId, framesEncoded</li>
         <li>RTCRemoteOutboundRTPStreamStats, with all required attributes from
         its inherited dictionaries, and also attributes localId, remoteTimestamp
         </li>
```

Comment:
Below this point: trackIdentifier, remoteSource, ended has moved from RTCMediaStreamTrackStats to RTCMediaHandlerStats. I think we should add "kind" as mandatory to RTCMediaHandlerStats.
frameWidth, frameHeight, framesPerSecond has moved to RTCVideoHandlerStats.
framesSent, has moved to RTCVideoSenderStats.
framesReceiverd, framesDecoded, framesDropped has moved to RTCVideoReceiverStats.
framesCorrupted seems to have disappeared - was it ever defined?
audioLevel has moved to RTCAudioHandlerStats.
RTCIceCaindidatePairStats.priority has been removed from RTCIceCandidatePairStats due to problems representing 64 significant bits in the 53 effective bits of "long long".

I think that's all.